AUTHORS

```diff
@@ -0,0 +1 @@
+Contributors to disq-bio:
```

Comment:
I recommend something with an explanation (sligthly modified from htsjdk-next-beta):

```
# This is the official list of DISQ authors for copyright purposes.
#
# The AUTHORS file lists the copyright holders.
# For example, employees from a company are not listed here, because the company holds the copyright.

# This file should be updated whenever a new contributor merges their first commit.

# Names should be added to this file as
# Organization <webpage> - Contact Person <email>
# Individual Name <email address or website>

# Please keep the list sorted alphabetically, companies/organizations first, individuals second.

Organizations:


Individuals:

```

Reply:
+1

Reply:
Hold up, I don't understand what this line means:
> For example, employees from a company are not listed here, because the company holds the copyright.

Reply:
So in htsjdk-next-beta, this line is specific to Broad Institute and it reads: "For example, Broad Institute employees are not listed here, because Broad Institute holds the copyright." I just tried to generalize, becasue no company is yet in the authors list. If you come up with a better phrasing, I'm fine with it too.

Reply:
I really really don't like this, to the point that I'm considering giving up on this whole effort.  I strongly believe the copyright has to be assigned to the project itself.

Reply:
What's the concern exactly?  If the code is licensed under MIT, it permits essentially anything except for relicensing under a more liberal license without any copyright assignment.

Reply:
The copyright is hold by the project itself and its authors, which are people who contributed to the project (we should decide if with code only or other types of contribution can apply).

A recognizition model in the AUTHORS file is a good way to encourage contributions.

Reply:
@heuermh I'm sorry you feel this way - I thought we were proceeding well so far since we had come to some consensus over copyright (e.g. https://github.com/disq-bio/disq/issues/8#issuecomment-406322856). Anyway, I hope you can reconsider.